test/sql/updates/post.continuous_aggs.v2.sql

```diff
@@ -12,7 +12,7 @@ CALL refresh_continuous_aggregate('cagg.realtime_mat',NULL,NULL);
 
 SELECT * FROM cagg.realtime_mat ORDER BY bucket, location;
 
-SELECT view_name, schedule_interval, materialized_only, materialization_hypertable_name FROM timescaledb_information.continuous_aggregates ORDER BY view_name::text;
+SELECT view_name, materialized_only, materialization_hypertable_name FROM timescaledb_information.continuous_aggregates ORDER BY view_name::text;
```

Comment:
this test should still compare the schedule_interval : select from continuous_aggregates view (pre 2.0).  b) Select schedule_interval from jobs table for 2.0+ releases.

Reply:
Since this is executed post update, we don't need 2 versions.  selecting schedule_interval for the policies from the jobs view should suffice.

Reply:
Added test for `schedule_interval`